hw/vendor/pulp_riscv_dbg/.gitignore

```diff
@@ -0,0 +1,2 @@
+.bender/
```

Comment:
Do we usually exclude such files from the vendoring? I guess not.

Reply:
We can instruct the vendor tool to exclude these files by enumerating the exclusion paths in the `*.vendor.hjson`:
https://github.com/lowRISC/opentitan/blob/bcab2e16acc40f1c6e0445e26b90f720717ea81d/hw/vendor/pulp_riscv_dbg.vendor.hjson#L20-L25